websites/A/AnimeLib/presence.ts

```diff
@@ -0,0 +1,352 @@
+import { AnimeLib } from "./lib";
+import type {
+	AnimeData,
+	UserData,
+	CharacterData,
+	PersonData,
+	CollectionData,
+	ReviewData,
+	TeamData,
+	PublisherData,
+} from "./lib";
+
+const presence = new Presence({
+		clientId: "1320289587943444552",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://i.imgur.com/8vAN1DA.png",
+	Play = "https://cdn.rcd.gg/PreMiD/resources/play.png",
+	Pause = "https://cdn.rcd.gg/PreMiD/resources/pause.png",
+}
+
+interface IFrameVideo {
+	duration: number;
+	currentTime: number;
+	paused: boolean;
+}
+
+let iFrameVideo: IFrameVideo, startTimestamp: number, endTimestamp: number;
+
+presence.on("iFrameData", (data: IFrameVideo) => {
+	iFrameVideo = data;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			type: ActivityType.Watching,
+			startTimestamp: browsingTimestamp,
+			smallImageText: "AnimeLIB",
+		},
+		strings = await presence.getStrings({
```

Comment:
I'm not sure why this should be used, honestly, since everything else is in Russian